salesforce_bulkipy/salesforce_bulkipy.py

```diff
@@ -1,15 +1,19 @@
 from __future__ import absolute_import
+from future.standard_library import install_aliases
+from future.utils import iteritems
+install_aliases()
+
 from tempfile import TemporaryFile
 from collections import namedtuple
 from httplib2 import Http
+from urllib.parse import urlparse
```

Comment:
Hello guys, this line is wrong, it should be:

```python
import urllib.parse as urlparse
```

Because in this same module, urlparse is accessed as a module and not a function, whereas in Python3 utllib.parse.urlparse is a function. This module accesses different functions in the module, such as `urljoin`.

Here is the module's documentation: https://docs.python.org/3.6/library/urllib.parse.html?highlight=urljoin#urllib.parse.urljoin